cpp/include/kvikio/shim/utils.hpp

```diff
@@ -23,6 +23,21 @@
 
 namespace kvikio {
 
+// Macros used for defining symbol visibility, only GLIBC is supported.
```

Comment:
and clang?

Reply:
Right, removed the `only GLIBC is supported` bit